<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>471</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Holocron Toolset</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QComboBox" name="gameCombo"/>
      </item>
      <item>
       <widget class="QTabWidget" name="resourceTabs">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="coreTab">
         <attribute name="title">
          <string>Core</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="coreSearchEdit">
            <property name="placeholderText">
             <string>search...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="coreTree">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="modulesTab">
         <attribute name="title">
          <string>Modules</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
            <item>
             <widget class="QComboBox" name="modulesCombo">
              <property name="maxVisibleItems">
               <number>18</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="moduleRefreshButton">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>70</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Refresh the list of modules.</string>
              </property>
              <property name="text">
               <string>Refresh</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="moduleSearchEdit">
              <property name="placeholderText">
               <string>search...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="moduleReloadButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Reload the active module.</string>
              </property>
              <property name="text">
               <string>Reload</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeView" name="modulesTree">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="overrideTab">
         <attribute name="title">
          <string>Override</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="overrideFolderFrame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="overrideFolderCombo"/>
             </item>
             <item>
              <widget class="QPushButton" name="overrideRefreshButton">
               <property name="minimumSize">
                <size>
                 <width>70</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Refresh the list of modules.</string>
               </property>
               <property name="text">
                <string>Refresh</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="overrideLine">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="overrideSearchEdit">
              <property name="placeholderText">
               <string>search...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="overrideReloadButton">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Refresh the list of files in the override folder.</string>
              </property>
              <property name="text">
               <string>Reload</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeView" name="overrideTree">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="texturesTab">
         <attribute name="title">
          <string>Textures</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QComboBox" name="texturesCombo"/>
          </item>
          <item>
           <widget class="Line" name="textureLine">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textureSearchEdit">
            <property name="placeholderText">
             <string>search...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TexturesView" name="texturesList">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="isWrapping" stdset="0">
             <bool>true</bool>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
            <property name="layoutMode">
             <enum>QListView::Batched</enum>
            </property>
            <property name="gridSize">
             <size>
              <width>92</width>
              <height>92</height>
             </size>
            </property>
            <property name="viewMode">
             <enum>QListView::IconMode</enum>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vboxlayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>124</width>
          <height>45</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="sidebar">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="openButton">
           <property name="text">
            <string>Open Selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="extractButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Extract Selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="tpcGroup_2">
           <property name="title">
            <string>TPC</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QCheckBox" name="tpcDecompileCheckbox">
              <property name="text">
               <string>Decompile</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="tpcTxiCheckbox">
              <property name="text">
               <string>Extract TXI</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="mdlGroup_2">
           <property name="title">
            <string>MDL</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QCheckBox" name="mdlDecompileCheckbox">
              <property name="text">
               <string>Decompile</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="mdlTexturesCheckbox">
              <property name="text">
               <string>Extract Textures</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>124</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNewDLG"/>
     <addaction name="actionNewNSS"/>
     <addaction name="actionNewUTC"/>
     <addaction name="actionNewUTP"/>
     <addaction name="actionNewUTD"/>
     <addaction name="actionNewUTI"/>
     <addaction name="actionNewUTS"/>
     <addaction name="actionNewUTT"/>
     <addaction name="actionNewUTM"/>
     <addaction name="actionNewUTW"/>
     <addaction name="actionNewUTE"/>
     <addaction name="separator"/>
     <addaction name="actionCloneModule"/>
     <addaction name="separator"/>
     <addaction name="actionNewGFF"/>
     <addaction name="actionNewERF"/>
     <addaction name="actionNewTXT"/>
     <addaction name="actionNewSSF"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="openAction"/>
    <addaction name="actionEditModule"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionEditTLK"/>
    <addaction name="actionEditJRL"/>
    <addaction name="separator"/>
    <addaction name="actionFileSearch"/>
    <addaction name="actionIndoorMapBuilder"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInstructions"/>
    <addaction name="actionHelpUpdates"/>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="cloneModuleAction">
   <property name="text">
    <string>Clone Module</string>
   </property>
  </action>
  <action name="actionRecent_Files">
   <property name="text">
    <string>Recent Files</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionCreature">
   <property name="text">
    <string>Creature</string>
   </property>
  </action>
  <action name="actionItem">
   <property name="text">
    <string>Item</string>
   </property>
  </action>
  <action name="actionDoor">
   <property name="text">
    <string>Door</string>
   </property>
  </action>
  <action name="actionPlaceable">
   <property name="text">
    <string>Placeable</string>
   </property>
  </action>
  <action name="actionMerchant">
   <property name="text">
    <string>Merchant</string>
   </property>
  </action>
  <action name="actionEncounter">
   <property name="text">
    <string>Encounter</string>
   </property>
  </action>
  <action name="actionTrigger">
   <property name="text">
    <string>Trigger</string>
   </property>
  </action>
  <action name="actionWaypoint">
   <property name="text">
    <string>Waypoint</string>
   </property>
  </action>
  <action name="actionDialog">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Dialog</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNewGFF">
   <property name="text">
    <string>GFF</string>
   </property>
  </action>
  <action name="actionNewERF">
   <property name="text">
    <string>ERF</string>
   </property>
  </action>
  <action name="actionNewTXT">
   <property name="text">
    <string>TXT</string>
   </property>
  </action>
  <action name="actionEditTLK">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit Talk Table</string>
   </property>
  </action>
  <action name="actionHelpUpdates">
   <property name="text">
    <string>Check For Updates</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNewSSF">
   <property name="text">
    <string>SSF</string>
   </property>
  </action>
  <action name="actionNewUTC">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/icons/kx/creature.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Creature</string>
   </property>
  </action>
  <action name="actionCloneModule">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clone Module</string>
   </property>
  </action>
  <action name="actionNewUTP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons/kx/placeable.png</normaloff>:/images/icons/kx/placeable.png</iconset>
   </property>
   <property name="text">
    <string>Placeable</string>
   </property>
  </action>
  <action name="actionNewUTD">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons/kx/door.png</normaloff>:/images/icons/kx/door.png</iconset>
   </property>
   <property name="text">
    <string>Door</string>
   </property>
  </action>
  <action name="actionNewUTW">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons/kx/waypoint.png</normaloff>:/images/icons/kx/waypoint.png</iconset>
   </property>
   <property name="text">
    <string>Waypoint</string>
   </property>
  </action>
  <action name="actionNewUTT">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons/kx/trigger.png</normaloff>:/images/icons/kx/trigger.png</iconset>
   </property>
   <property name="text">
    <string>Trigger</string>
   </property>
  </action>
  <action name="actionNewUTE">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons/kx/encounter.png</normaloff>:/images/icons/kx/encounter.png</iconset>
   </property>
   <property name="text">
    <string>Encounter</string>
   </property>
  </action>
  <action name="actionNewUTS">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons/kx/sound.png</normaloff>:/images/icons/kx/sound.png</iconset>
   </property>
   <property name="text">
    <string>Sound</string>
   </property>
  </action>
  <action name="actionNewUTM">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons/kx/merchant.png</normaloff>:/images/icons/kx/merchant.png</iconset>
   </property>
   <property name="text">
    <string>Merchant</string>
   </property>
  </action>
  <action name="actionNewUTI">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons/kx/item.png</normaloff>:/images/icons/kx/item.png</iconset>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </action>
  <action name="actionNewDLG">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons/kx/dialog.png</normaloff>:/images/icons/kx/dialog.png</iconset>
   </property>
   <property name="text">
    <string>Dialog</string>
   </property>
  </action>
  <action name="actionNewNSS">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons/kx/script.png</normaloff>:/images/icons/kx/script.png</iconset>
   </property>
   <property name="text">
    <string>Script</string>
   </property>
  </action>
  <action name="actionEditJRL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit Journal</string>
   </property>
  </action>
  <action name="actionFileSearch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>File Search</string>
   </property>
  </action>
  <action name="actionGeometryEditor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Geometry Editor</string>
   </property>
  </action>
  <action name="actionIndoorMapBuilder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Indoor Map Builder</string>
   </property>
  </action>
  <action name="actionEditModule">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit Module</string>
   </property>
  </action>
  <action name="actionInstructions">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TexturesView</class>
   <extends>QListView</extends>
   <header location="global">toolset.mainwindow</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
